src/main/java/Ui.java

```diff
@@ -0,0 +1,130 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    // this field is used when output requires a horizontal line
+    public static final String LINE = "    ____________________________________________________________";
+
+    // this function greets the user when Duke is started
+    public void greet(String createResult) {
```

Comment:
I think for some of the methods used by Ui should be changed to verbs form, such as `printGreetingMessage`.